config.example.yaml

```diff
@@ -56,3 +56,6 @@ council_changelog: 1234567890
 # Voting parameters
 required_difference: 5  # Majority required for this blob to move into the next stage
 required_votes: 15  # Minimum amount of total votes before moving to next stage
+
+# Versus type
+verbose_vs: false
```

Comment:
An explanation of what this means should be included a la the other config values.